cmd/hook.go

```diff
@@ -49,13 +48,13 @@ func launchHook(hook string, timeout time.Duration, meta map[string]string) erro
 		return fmt.Errorf("start command: %w", err)
 	}
 
-	timer := time.AfterFunc(timeout, func() {
-		log.Println("hook timed out: killing command")
-		_ = cmd.Process.Kill()
-		_ = stdout.Close()
-	})
-
-	defer timer.Stop()
+	go func() {
+		<-ctxCmd.Done():
```

Comment:
my bad, this ought to be

```suggestion
		<-ctxCmd.Done()
```

Reply:
fixes :wink: